packages/graphics/src/GraphicsData.ts

```diff
@@ -1,3 +1,13 @@
+import { Circle, Ellipse, Rectangle, Polygon, RoundedRectangle, Matrix, SHAPES } from '@pixi/math/src';
+import { FillStyle } from './styles/FillStyle';
+import { LineStyle } from './styles/LineStyle';
+
+/**
+ * @description Complex shape type
+ * @todo Move to Math shapes
+ */
+type IShape = Circle | Ellipse | Polygon | Rectangle | RoundedRectangle;
```

Comment:
Should this be moved to math then?

Reply:
Yes, i think it must be moved to math then

Reply:
done